src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto;
+}
+
+.card__title {
+  display: block;
+  padding: 40px 0 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
```

Comment:
you can put this color to reset and don't use in other parts, or better to make a variable

Reply:
good point, done